controller-server/src/main/java/com/yahoo/vespa/hosted/controller/deployment/InternalStepRunner.java

```diff
@@ -702,10 +706,42 @@ private void sendNotification(Run run, DualLogger logger) {
             mailOf(run, recipients).ifPresent(controller.serviceRegistry().mailer()::send);
         }
         catch (RuntimeException e) {
-            logger.log(INFO, "Exception trying to send mail for " + run.id(), e);
+            logger.log(WARNING, "Exception trying to send mail for " + run.id(), e);
         }
     }
 
+    private void updateConsoleNotification(Run run) {
+        NotificationSource source = NotificationSource.from(run.id());
+        Consumer<String> updater = msg -> controller.notificationsDb().addNotification(source, Notification.Type.DEPLOYMENT_FAILURE, msg);
+        switch (run.status()) {
+            case running:
+            case aborted:
+                return; // If running, its too early to update. If aborted, let's wait and see how the next run goes.
+            case success:
+                controller.notificationsDb().removeNotification(source, Notification.Type.DEPLOYMENT_FAILURE);
+                return;
+            case outOfCapacity:
+                if (run.id().type().isProduction()) updater.accept("due to lack of capacity. Please contact the Vespa team to request more!");
```

Comment:
```suggestion
                if ( ! run.id().type().environment().isTest()) updater.accept("due to lack of capacity. Please contact the Vespa team to request more!");
```
No warnings for dev/perf?